src/index.html

```diff
@@ -5,9 +5,43 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section class="product-card" data-qa="card">
+
+      <img
+        src="images/imac.jpeg"
+        class="product-card__img"
+        alt="iMac">
+
+      <h2 class="product_name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      <p class="product_code">Product code: 195434</p>
+      <div class="stars-and-reviews">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="product_reviews">Reviews: 5</div>
+      </div>
+
+      <div class="price">
+        <div class="price__name">Price:</div>
+        <div class="price__value">$2,199</div>
+      </div>
+      <a
+      href="#"
+      class="product-card__button"
+      data-qa="hover"
+      >
```

Comment:
Pay attention on indents
```suggestion
      <a
        href="#"
        class="product-card__button"
        data-qa="hover"
      >
```